coral-component-table/src/scripts/Table.js

```diff
@@ -2370,8 +2397,15 @@ class Table extends BaseComponent(HTMLTableElement) {
   _setHeaderCellScope(headerCell, tableSection) {
     // Add appropriate scope depending on whether header cell is in THEAD or TBODY
     const scope = tableSection.nodeName === 'THEAD' || tableSection.nodeName === 'TFOOT' ? 'col' : 'row';
-    const ariaRole = scope === 'col' ? 'columnheader' : 'rowheader';
-    headerCell.setAttribute('role', ariaRole);
+    if(scope === 'col') {
+      if(this._isTableSticky() === true) {
```

Comment:
you don't need === true IMO and please add space after if

Reply:
done